### ISSUE TITLE
Jbrowse-cli broken in release 0.0.1-beta.19

### ISSUE THREAD

**Issue by Akusem** [GMOD/jbrowse-components#1187]
**Describe the bug**
When trying yo use the new version version of Jbrowse-cli, I get errors about the commands not existing.
Example when using the command `jbrowse create ./jbrowse`:
```
    Error: Cannot find module '/path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/lib/commands/create'
    Require stack:
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/plugin.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/config.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/index.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/command/lib/command.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/command/lib/index.js
    - /path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/bin/run
    Code: MODULE_NOT_FOUND
```
This error is present with all the cli command like add-track or add-assembly.

I've taken a look in /.config/yarn/global/node_modules/@gmod/jbrowse-cli/ and the folder `lib/` is not present. However in the last version working, 0.0.1-beta.17, it was here. 

**To Reproduce**

Describe here the steps to reproduce the behavior, e.g.:
1. Install the last version of jbrowse-cli: `yarn global add @gmod/jbrowse-cli@0.0.1-beta.19`
2. Try to use jbrowse-cli: `jbrowse create ./test_jbrowse`

**Version:**

jbrowse-cli version 0.0.1-beta.19

**Additional Information**
Directory structure of jbrowse-cli downloaded by yarn
```
tree /path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/
├── bin
│   ├── run
│   └── run.cmd
├── CHANGELOG.md
├── oclif.manifest.json
├── package.json
└── README.md
```

**Comment by cmdcolin** [comment]
Can confirm. Thanks for checking this. Currently doing some stuff to help automated builds get deployed but it's unexpected that jbrowse-cli broke!

**Comment by cmdcolin** [comment]
Current status is that `yarn tsc` or `yarn tsc -b` produces no output

Any ideas on this @garrettjstevens ? It is kinda weird, the outDir is not created from tsc even though tsc "passes"

**Comment by cmdcolin** [comment]
Possible fix in #1189

**Comment by cmdcolin** [comment]
@Akusem this issue in the CLI should be fixed now, reinstall npm install -g @gmod/jbrowse-cli and get the latest (currently @gmod/jbrowse-cli@0.0.1-beta.21)

Thanks so much for reporting
